shared/templates/static/bash/file_permissions_sshd_pub_key.sh

```diff
@@ -1,3 +1,8 @@
 # platform = multi_platform_rhel
 
-chmod 0644 /etc/ssh/*.pub
+#check if files exist before attempting 
+#to change them
+if ls /etc/ssh/*.pub &> /dev/null
+then
+    chmod 0644 /etc/ssh/*.pub
+fi
```

Comment:
Please change all of this to just `chmod -f /etc/ssh/*.pub`.

Although I am wondering if this should fail as this key should exist.